### PR TITLE
Separate change kinds

### PR DESCRIPTION
**Pull request by kitagry**
## before

```markdown
* changed
  * apps.v1.Deployment.default.hoge
  * apps.v1.Deployment.default.fuga
  * apps.v1.Deployment.default.piyo
```

## after

```markdown
## created
* apps.v1.Deployment.default.hoge
## pruned
* apps.v1.Deployment.default.fuga
## configured
* apps.v1.Deployment.default.piyo
```

### CODE REVIEW

File: Cargo.toml
Comment (hirosassa) [inline Cargo.toml:1]: Thanks!

**Review by hirosassa — APPROVED** [review]
Looks great!

File: src/template.rs [hunk 1]
```diff
@@ -25,8 +27,24 @@ impl Template {
...
+* pruned
+{{#each pruned_kinds}}
+  * {{this}}
+{{/each}}
+{{/if}}
+{{#if (gt (len configured_kinds) 0)}}
+* configured
+{{#each configured_kinds}}
+  * {{this}}
+{{/each}}
```
Comment (hirosassa) [inline src/template.rs]: [comment] @kitagry 
Did you consider using H2 (or smaller one) as a separator?
If not, I think it is good to compare the format.

```suggestion
{{#if (gt (len created_kinds) 0)}}
## created
{{#each created_kinds}}
  * {{this}}
{{/each}}
{{/if}}
{{#if (gt (len pruned_kinds) 0)}}
## pruned
{{#each pruned_kinds}}
  * {{this}}
{{/each}}
{{/if}}
{{#if (gt (len configured_kinds) 0)}}
## configured
{{#each configured_kinds}}
  * {{this}}
{{/each}}
```

**Comment by kitagry** [comment]
I fixed!

**Comment by hirosassa** [comment]
@kitagry Thanks!

**Pull request merged**